test/cypress/plugins/index.ts

```diff
@@ -0,0 +1,22 @@
+/// <reference types="cypress" />
+// ***********************************************************
+// This example plugins/index.js can be used to load plugins
+//
+// You can change the location of this file or turn off loading
+// the plugins file with the 'pluginsFile' configuration option.
+//
```

Comment:
Are the files in the test/cypress folder necessary? They don't seem to be doing anything

Reply:
This file was autogenerated, so I did not touch it.